content/applications/marketing/events/create_events.rst

```diff
@@ -0,0 +1,374 @@
+
+For :guilabel:`Selection` amd :guilabel:`Text Input` types, a clickable :guilabel:`📊 Stats` icon
+appears on the right side of the question line. When clicked, Odoo reveals a separate page,
+showcasing the response metrics to that specific question.
+
+To delete any question from the :guilabel:`Questions` tab, simply click the :guilabel:`🗑️ (trash
+can)` icon on the corresponding question line.
+
+There is no limit to how many questions can be added in the :guilabel:`Questions` tab of an event
+form.
+
+.. _create_events/event_form/notes:
+
+Notes tab
+---------
+
+In the :guilabel:`Notes` tab of an event form, users can leave detailed internal notes and/or
+event-related instructions/information for attendees.
+
+.. image:: create_events/notes-tab.png
+   :align: center
+   :alt: Typical notes tab on an event form in the Odoo Events application.
+
+In the :guilabel:`Note` field of the :guilabel:`Notes` tab, users can leave internal notes for other
+event employees, like "to-do" lists, contact information, instructions, and so on.
+
+In the :guilabel:`Ticket Instructions` field of the :guilabel:`Notes` tab, users can leave specific
+instructions for people attending the event, like how to enter, when the venue closes, and so on.
+
+Publish events
+==============
+
+Once all configurations and modifications are complete on the event form, it is time to publish the
+event on the website. Doing so makes the event visible to website visitors, and makes it possible
+for interested parties to register for the event.
+
+To publish an event after all the customizations are complete, click the :guilabel:`Go to Website`
+smart button at the top of the event form. Doing so reveals a blank event web page, which can be
+customized like any other web page on the site, via the :guilabel:`Edit` button.
+
+To learn more about website design functionality and options, consult the :doc:`Building block
+<../../websites/website/web_design/building_blocks>` documentation.
+
+Once the event website is ready to be shared, click the red :guilabel:`Unpublished` toggle switch
+in the header menu, changing it to a green :guilabel:`Published` switch. At this point, the event
+web page is published, and viewable/accessible by all website visitors.
+
+Send event invites
+==================
+
+To send event invites to potential attendees, navigate to the desired event form, via
+:menuselection:`Events app --> Events`, and click the :guilabel:`Invite` button in the upper-left
+corner of the event form.
+
+Doing so reveals a blank email form to fill out, as desired. To learn more about how to create and
+customize emails like this, refer to the :ref:`Create an email <email_marketing/create_email>`
+documentation.
+
+Proceed to create and customize an email message to send as an invite to potential attendees.
+Remember to include a link to the registration page on the event website, allowing recipients to
+quickly register, if they're interested.
+
```

Comment:
.. tip::
   Sending emails from Odoo is subject to a daily limit, which by default is 200. To learn more about daily limits          visit this documentation: :ref:`email_communication/daily_limit_mail`.